app/src/main/java/com/mymeetings/android/db/Daos.kt

```diff
@@ -6,17 +6,17 @@ import androidx.room.*
 interface CalendarEventsDao {
```

Comment:
The name of the file should reflect the interface contained within, which is `CalendarEventsDao.kt`.

Reply:
Let group all the daos under one kt file instead of creating multiple files.

Reply:
No, it is difficult to search for files, yesterday I had to spend 20 minutes searching for the dao file. Let's organize files cleanly to later find the codebase readable. Please do not group more than one class / interface into single files.